cmd/go-quai/usage.go

```diff
@@ -145,8 +145,6 @@ var AppHelpFlagGroups = []flags.FlagGroup{
 	{
 		Name: "MINER",
 		Flags: []cli.Flag{
-			utils.MiningEnabledFlag,
-			utils.MinerThreadsFlag,
 			utils.MinerNotifyFlag,
```

Comment:
do we need these other flags?

Reply:
I've removed the unused mining related flags since your comment.